docs/assertions/assertions.mdx

```diff
@@ -172,6 +172,8 @@ $actual | Should -BeOfType System.IO.FileSystemInfo  # Test will pass; Directory
 $actual | Should -BeOfType System.IO.FileInfo        # Test will fail; FileInfo is not a base class of DirectoryInfo
 ```
 
+**Note:** This currently only works for .NET types. For custom type name, added using PSTypeNames, you need to use `$MyClass.GetType().Name | Should -Be "MyClassName"`. See [this issue](https://github.com/pester/Pester/issues/1315) for more information.
```

Comment:
Could you update to use note [admonition](https://docusaurus.io/docs/markdown-features/admonitions)?

Reply:
> Not sure why this hasn't been merged yet, but since the styling has changed recently, would you mind a minor update? 😇

I was basically for someone to review or thumbs-up

Reply:
Oh.. I considered the review from @nohwnd that he resolved himself a thumbs up.
Now we can wait though 😄

Reply:
Totally missed that of course lol 💃